src/sidebar/websocket.js

```diff
@@ -109,7 +120,20 @@ export default class Socket extends EventEmitter {
 
     /** Close the underlying WebSocket connection */
     this.close = function() {
-      socket.close();
+      // nb. h's ws4py-based WebSocket server will currently use whatever
```

Comment:
This expanded comment is great! Might consider starting it with a quick summary for the tl;dr of us, e.g. "Always close with a normal status code to work around a problem with h's ws4py package"